api/api_tests.py

```diff
@@ -591,17 +584,25 @@ def test_000_user(self):
         self.assertEqual(response['user']['contribution_percent'], 1)
         self.assertEqual(response['user']['has_items'], False)
 
+        code, response = self.update_user(token_1, False, email="brokenemail")
+        self.assertEqual(code, 400)
+        self.assertIn("invalid", response['message'])
+
+        code, response = self.update_user(token_1, False, email="brokenemail@plebinexistingdomain.com")
+        self.assertEqual(code, 400)
```

Comment:
Why is this invalid?

Reply:
"inexistingdomain" is part of the domain name :)

this validator actually checks for the domain.